### DISCUSSION TITLE
Adjust default label padding on MacOS platform

### DISCUSSION THREAD

**Discussion by InfinityGhost** [picoe/Eto#1844] [Q&A] [answered]
The padding on labels on MacOS is excess for my current use case in a custom control. I want it to closer match my GTK app's design if at all possible. Is anyone aware of a way to modify it? (Using 0 padding on both)

## MacOS
![macos](https://user-images.githubusercontent.com/39218853/101712928-953ee600-3a64-11eb-9c6f-9a442201e654.png)

## GTK
![gtk](https://user-images.githubusercontent.com/39218853/101712904-8eb06e80-3a64-11eb-860e-2071de91aadb.png)

If anyone's interested in the source code to see if they can see any obvious issue with my implementation: https://github.com/InfinityGhost/OpenTabletDriver/pull/531

**Comment by cwensley** [comment, marked as answer]
Hm, I think the biggest issue you might be facing is the default padding for the GroupBox.  That, and the width of the numeric entry boxes are relatively larger on macOS (~9 characters wide vs ~7 with Gtk). Unfortunately, Eto's GroupBox sizing calculations are fixed, so adjusting the padding isn't really possible with 2.5.8.  That being said, I've just committed a fix in #1858 that allows you to add something like this to adjust the padding/content margins in a GroupBox on macOS:

```
	Eto.Style.Add<Eto.Mac.Forms.Controls.GroupBoxHandler>(null, h => {
		h.Control.ContentViewMargins = new CGSize(2, 2);
	});
```

Hope this helps!
